tests/data/mock_manifests/ValidFilenameManifest.csv

```diff
@@ -1,5 +1,5 @@
-Component,Filename,Id,entityId
-MockFilename,schematic - main/TestSubmitMockFilename/txt1.txt,3c4c384b-5c49-4a7c-90a6-43f03a5ddbdc,syn62822369
-MockFilename,schematic - main/TestSubmitMockFilename/txt2.txt,3b45f5f3-408f-47ff-945e-9badf0a43195,syn62822368
-MockFilename,schematic - main/TestSubmitMockFilename/txt3.txt,2bbc898f-2651-4af3-834a-10c506de0fbd,syn62822366
-MockFilename,schematic - main/TestSubmitMockFilename/txt4.txt,5a2d3816-436e-458f-9887-cb8355518e23,syn62822364
+Component,SampleID,Filename,Id,entityId
```

Comment:
here, since the component is `MockFilename`, I am fine with this column be kept as "SampleID".